sdk/storage/Azure.Storage.Queues/api/Azure.Storage.Queues.netstandard2.0.cs

```diff
@@ -73,6 +82,11 @@ public enum ServiceVersion
             V2020_02_10 = 4,
         }
     }
+    public enum QueueMessageEncoding
+    {
+        UTF8 = 0,
+        Base64 = 1,
```

Comment:
I find these names a little confusing.  Isn't the Base64 encoded value still sent as UTF8?  Should be it `None` and `Base64`?

Reply:
Done.